docs/advanced-usage.rst

```diff
@@ -216,89 +335,197 @@ If we try to create another transaction with the same input as before, the
+        },
+        'owners_before': condition['owners_after'],
+    }
+
+Let's take a moment to contemplate what this ``input_`` is:
+
+.. code-block:: python
+
+    >>> input_
+    {'fulfillment': {'bitmask': 41,
+      'subfulfillments': [{'bitmask': 32,
+        'public_key': '3Cxh1eKZk3Wp9KGBWFS7iVde465UvqUKnEqTg2MW4wNf',
+        'signature': None,
+        'type': 'fulfillment',
+        'type_id': 4,
+        'weight': 1},
+       {'bitmask': 32,
+        'public_key': 'EcRawy3Y22eAUSS94vLF8BVJi62wbqbD9iSUSUNU9wAA',
+        'signature': None,
+        'type': 'fulfillment',
+        'type_id': 4,
+        'weight': 1}],
+      'threshold': 2,
+      'type': 'fulfillment',
+      'type_id': 2},
+     'input': {'cid': 0,
+      'txid': '3128572d430f9ae23010f11e444d1d12e35f90095cf5f0da98acfcada5eed0fe'},
+     'owners_before': ['3Cxh1eKZk3Wp9KGBWFS7iVde465UvqUKnEqTg2MW4wNf',
+      'EcRawy3Y22eAUSS94vLF8BVJi62wbqbD9iSUSUNU9wAA']}
+
+and the asset:
+
+.. code-block:: python
+
+    asset = signed_car_creation_tx['transaction']['asset']
+
+then ``alice`` can prepare the transfer:
+
+.. code-block:: python
+
+    car_transfer_tx = bdb.transactions.prepare(
+        operation='TRANSFER',
+        owners_after=carol.verifying_key,
+        asset=asset,
+        inputs=input_,
+    )
+
+The asset can be transfered as soon as each of the ``owners_after`` fulfills
+the transaction, that is ``alice`` and ``bob``.
+
+To do so, simply provide a list of all private keys to the fulfill method:
+
+.. code-block:: python
+
+    signed_car_transfer_tx = bdb.transactions.fulfill(
+        car_transfer_tx, private_keys=[alice.signing_key, bob.signing_key]
+    )
+
+Note, that if one the signing keys is missing, the fulfillment will fail. If we
+omit ``bob``:
+
+.. code-block:: python
+
+    try:
+        signed_car_transfer_tx = bdb.transactions.fulfill(
+            car_transfer_tx,
+            private_keys=alice.signing_key,
+        )
+    except MissingSigningKeyError as e:
+        print(e, e.__cause__, sep='\n')
+
+You should see a message similar to:
+
+.. code-block:: text
+
+    A signing key is missing!
+    Public key EcRawy3Y22eAUSS94vLF8BVJi62wbqbD9iSUSUNU9wAA is not a pair to any of the private keys
+
+Notice that is ``bob``'s public key in the above message:
+
+.. code-block:: python
+
+    >>> bob.verifying_key
+    'EcRawy3Y22eAUSS94vLF8BVJi62wbqbD9iSUSUNU9wAA'
+
+Same goes for ``alice``. Try it!
+
+Sending the transaction over to a BigchainDB node:
+
+.. code-block:: python
+
+    sent_car_transfer_tx = bdb.transactions.send(signed_car_transfer_tx)
+
+if ``alice`` and ``bob`` wish to check the status of the transfer they may use
+the :meth:`~bigchaindb_driver.BigchainDB.transactions.status` endpoint:
+
+.. code-block:: python
+
+    >>> bdb.transactions.status(sent_car_transfer_tx['id'])
+    {'status': 'valid'}
+
+Done!
+
+Happy, ``alice`` and ``bob`` have successfully transferred the ownership of
```

Comment:
It just reads a little odd, but yes, generally criticism on writing is subjective.